include/mbedtls/ecdh.h

```diff
@@ -187,7 +199,7 @@ void mbedtls_ecdh_free( mbedtls_ecdh_context *ctx );
  * \brief           This function generates a public key and a TLS
  *                  ServerKeyExchange payload.
  *
- *                  This is the first function used by a TLS server for ECDHE
+ *                  This is the second function used by a TLS server for ECDHE
```

Comment:
Please add a reference to the first function or to a place where all the successive functions are listed.